src/components/usersTable/usersTable.js

```diff
@@ -0,0 +1,37 @@
+import {Table, TableBody, TableCell, TableContainer, TableHead, TableRow} from "@material-ui/core";
+
+
+export default function UsersTable({
+  users
+}) {
+  return (
+    <TableContainer>
+      <Table sx={{minWidth: 700}} aria-label="users table">
+        <TableHead>
+          <TableRow>
+            <TableCell>Name</TableCell>
+            <TableCell alignRight>Email</TableCell>
```

Comment:
There is no `alignRight` property but `align="right"`. Anyway, I would not change default left align as it looks more correct.